azure-pipelines.yml

```diff
@@ -29,8 +34,16 @@ stages:
       displayName: 'Install Node.js 10.16.3'
     - script: yarn install
       displayName: 'Run yarn install'
+    - script: yarn test
+      displayName: 'Run yarn test'
```

Comment:
Is there a reason we are running this 3 times?

Reply:
I've just always run my tests on each OS in the matrix - we can just run once if you'd like to pick an OS to be the "test runner" - that would also mean that we wouldn't necessarily need puppeteer either as that's mainly for MacOS as the AzDo image doesn't have chrome installed.

Reply:
@frozenskys: The assumption for tests is normally that they are platform independent, so there should not be any need to run them on different platforms. In a way I am trying to be economical in the use of the free resource that we are using. If Chrome is available on Windows then running the tests on Windows makes sense, as this is by far the most popular platform.

Reply:
@mikeller makes sense, also windows has chrome pre-installed so will simplify things a little as we won't need puppeteer. Speaking of chrome I see there has been some discussion around the chromeos build missing, did we want to add it back in?

Reply:
Not at the moment. We are considering dumping support for the Chrome app build altogether, as it is severely hampering our development by limiting our choices of frameworks / packages to whatever is web ready and will work with the hardcoded CSP of Chrome apps, and it's only actually needed by 0.3% of our active user base. So adding this to the new build pipeline seems to be moot before we know that we will keep supporting it.